shared/game.js

```diff
@@ -21,7 +21,7 @@ var Game = Interface.define("Game", {
           setSystemBySymbol = systems.set;
       systems.getBySymbol = getSystemBySymbol;
       systems.get = (function systems$get$(interface, ent) {
-        /* systems.get eval.sibilant:1:831 */
+        /* systems.get eval.sibilant:1:862 */
```

Comment:
Think this is from an update to the compiler where we added an include.

Reply:
Either that or again because we changed the order and we're using sibilant in a very strange way.

Reply:
We should look at the sibilant source code, probably pull it into this repo, and find out where it does this and grok it. I've read it before, it was pretty straight forward iirc

Reply:
Added a task for this